### ISSUE TITLE
Japanese im input problems

### ISSUE THREAD

**Issue by whip-cream** [marzent/XIV-on-Mac#164]
When entering Japanese, some characters are entered in duplicate, or previously made statements are re-entered.

Only multibyte characters cause problems, alphanumeric character examples do not

thank you.

**Comment by marzent** [comment]
Yep this is a known problem being worked on, but thanks for pointing it out!
